pkg/testutils/serverutils/test_server_shim.go

```diff
@@ -31,15 +34,60 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/security/username"
 	"github.com/cockroachdb/cockroach/pkg/server/status"
 	"github.com/cockroachdb/cockroach/pkg/storage"
+	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
 	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
 	"github.com/cockroachdb/cockroach/pkg/util/hlc"
 	"github.com/cockroachdb/cockroach/pkg/util/httputil"
+	"github.com/cockroachdb/cockroach/pkg/util/log"
 	"github.com/cockroachdb/cockroach/pkg/util/protoutil"
 	"github.com/cockroachdb/cockroach/pkg/util/stop"
 	"github.com/cockroachdb/cockroach/pkg/util/uuid"
 	"github.com/cockroachdb/errors"
 )
 
+var tenantModeFlag = flag.String(
+	"tenantMode", "default",
+	"tenantMode in which to run tests. Options are forceTenant, forceNoTenant, and default "+
+		"which alternates between tenant and no-tenant mode probabilistically. Note that the two force "+
```

Comment:
Could you make the default overridable with a bool var (say `COCKROACH_TESTING_TENANT_MODE`, `envutil` has methods for this stuff.

The rationale is that if we want to use our CI jobs to stress roachtests, it can be annoying to work with a flag. The flag as defined here is not necessarily present for each test package (since not all packages import `serverutils`), and it is more convenient to set an env var.